src/Compilers/CSharp/Portable/BoundTree/Formatting.cs

```diff
@@ -163,4 +163,9 @@ internal partial class BoundPassByCopy
     {
         public override object Display => Expression.Display;
     }
+
+    internal partial class UnboundObjectCreationExpression
+    {
+        public override object Display => "new()";
```

Comment:
>"new()"; [](start = 42, length = 8)

nit: should we display the arguments too? or maybe put an ellipsis `new(...)`

Reply:
We have three options:

1. always `new()` (current iteration)
1. always `new(...)` (previous iteration)
1. conditionally `new()` if there's no args, otherwise `new(...)`

I didn't like (2) because I suspect we'll have `new()` most of the time. I'm fine with other two, I went with (1) for now to keep things simple.

Reply:
I think it's fine to leave as-is (option 1). Thanks

Reply:
Given that this is used in error output, I don't think that this is fine. We should do option 3, and correctly display the arguments.

Reply:
@alrz I didn't see any response to this?